elements/pfe-jump-links/pfe-jump-links-nav.ts

```diff
@@ -38,13 +38,16 @@ function isQueryable(x: Node): x is Document|ShadowRoot {
 
 /**
  * @fires pfe-jump-links-panel:active-navItem
```

Comment:
can we deprecate the el:name events and add our own `extends Event` events?

Reply:
Good idea.  Just updated them to our new `ComponedEvent`